transformers/tables.go

```diff
@@ -28,3 +29,19 @@ func TransformTables(tables schema.Tables) error {
 	}
 	return nil
 }
+
+// ApplyTransformers applies the given transformers to the given Tables, and recursively on the relations.
+// This is useful for applying transformers that are not defined in the table definitions. To apply the table-definition transformers, use TransformTables.
+func ApplyTransformers(tables schema.Tables, extraTransformers ...schema.Transform) error {
```

Comment:
```suggestion
func Apply(tables schema.Tables, extraTransformers ...schema.Transform) error {
```

Because it will be called as `transformers.Apply(tables, ...)`

Reply:
Could we add a test for this function? :)

Reply:
good catch :)